cmd/rokka/cli/sourceimages.go

```diff
@@ -8,12 +8,15 @@ import (
 	"os"
 	"path"
 
+	"github.com/rokka-io/rokka-go/cmd/rokka/cli/copyall"
 	"github.com/rokka-io/rokka-go/rokka"
 	"github.com/spf13/cobra"
+	"gopkg.in/cheggaaa/pb.v1"
```

Comment:
could you use [`dep ensure`](https://github.com/golang/dep) to add this dependency?

Reply:
done